core/state/trie_prefetcher.go

```diff
@@ -165,20 +168,61 @@ func (p *triePrefetcher) copy() *triePrefetcher {
 	}
 	return copy
 }
+func (p *triePrefetcher) cpy4Prefetcher() *triePrefetcher {
+	cpy := &triePrefetcher{
+		db:           p.db,
+		root:         p.root,
+		fetchers:     p.fetchers,
```

Comment:
the map will have concurrent access issue.
read by several `triePrefetcher` in different routine, and updated in prefetchLoop.

Reply:
won't have concurrency access issue, access would be managed by a shared channel with the same loop

Reply:
yes, you are right